drivers/network/tcpip/tcpip/iinfo.c

```diff
@@ -83,9 +90,12 @@ TDI_STATUS InfoTdiQueryGetInterfaceMIB(TDIEntityID ID,
                     case NdisPhysicalMedium1394:
                     case NdisPhysicalMediumWirelessWan:
 #endif
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
```

Comment:
Shouldn't a similar `#if` be added to the definition too?
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=NdisPhysicalMediumNative802_11
```
sdk/include/psdk/ntddndis.h	
 216     NdisPhysicalMediumNative802_11,
```